api/client/trust.go

```diff
@@ -372,21 +372,30 @@ func (cli *DockerCli) trustedPush(repoInfo *registry.RepositoryInfo, tag string,
 
 	defer responseBody.Close()
 
-	targets := []target{}
+	// If it is a trusted push we would like to find the target entry which match the
+	// tag provided in the function and then do an AddTarget later.
```

Comment:
Can you please add to the comment that trusted push only supports pushing one tag at a time, so we don't attempt to handle multiple target entries in the push output?